helm/g2p-sandbox/values.yaml

```diff
@@ -490,3 +492,51 @@ ph-ee-engine:
       ingress:
         enabled: false
         hostname: admin-kong.sandbox.mifos.io
+
+  account_mapper:
+    enabled: true
+    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/ph-ee-vouchers"
```

Comment:
please point to the docker hub images
why the voucher image is used here in account mapper?